### ISSUE TITLE
userStory_05 -EditWeeklyOverview

### ISSUE THREAD

**Issue by Bjoern-Jentschke** [gcode-de/mealwheel#5]
## Value proposition
As a user 
I want to change the meals manually or the number of persons eating
In order to customise the meals to match my needs.
## Description

![Image](https://github.com/gcode-de/mealwheel/assets/155737121/870c0f46-8b37-46d5-819a-2ee092878f84)

## Acceptance criteria
- next on the right hand side to each day and meal there is now one button to increase the number of persons joining to eat and another button to change the recipe randomly
- there is a counter on the "add person" button that displays the number of people joining
## Tasks
- add the "add person button"
- add a person counter to the button
- add the "change recipe button"
- after clicking the "change recipe button" a new random recipe will be displayed for the underlying day
- the quantity of ingredients has to change proportionally to the number of people joining

**Comment by gcode-de** [comment]
- [ ] the quantity of ingredients has to change proportionally to the number of people joining
this is more relevant on the recipe details page and the shopping list, right?

**Comment by Bjoern-Jentschke** [comment]
> * [ ]  the quantity of ingredients has to change proportionally to the number of people joining
>   this is more relevant on the recipe details page and the shopping list, right?

Yes! I added the information to the tasks, also pointing out that there is no shopping list yet

**Comment by gcode-de** [comment]
Additional tasks required:

- [ ] add functions to create, read and update calendar entries from user object and display them in the UI
